vizro-core/src/vizro/models/_components/graph.py

```diff
@@ -37,39 +38,17 @@ class Graph(VizroBaseModel):
     """
 
     type: Literal["graph"] = "graph"
-    figure: CapturedCallable = Field(..., import_path=px)
+    figure: CapturedCallable = Field(
+        ..., import_path=px
+    )  # TODO: needs to be refactored so plotly-independent or extendable - No, as within the boundaries of Graph model
```

Comment:
```suggestion
    figure: CapturedCallable = Field(..., import_path=px)
```

I believe this is resolved?

Reply:
Yes